cartridges/app_storefront_custom/cartridge/templates/default/showVariable/test.isml

```diff
@@ -0,0 +1,8 @@
+<isdecorate template="common/layout/page">
+    <isif condition="${session.custom.x === 1}">
```

Comment:
Isn't this if/else logic only going to increment x to 2, and then start again?

Reply:
Fixed it